connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/processors/recipients/RecipientsResolverResponseProcessor.java

```diff
@@ -28,8 +37,20 @@ public class RecipientsResolverResponseProcessor implements Processor {
     @Override
     public void process(final Exchange exchange) throws JsonProcessingException {
         final String body = exchange.getMessage().getBody(String.class);
-        final List<User> recipientsList = Arrays.asList(this.objectMapper.readValue(body, User[].class));
+        final Set<String> recipientsList = new HashSet<>(Arrays.asList(this.objectMapper.readValue(body, User[].class)))
+            .stream().map(User::getEmail).collect(toSet());
+
+        Set<String> emails = exchange.getProperty(ExchangeProperty.EMAIL_RECIPIENTS, Set.class);
+        recipientsList.addAll(emails);
+
+        // We have to remove one from the limit, because a default recipient (like noreply@redhat.com) will be automatically added
```

Comment:
@gwenneg just to point this out, I remember that we use a default recipient which consume one place ;)